Makefile

```diff
@@ -132,11 +132,15 @@ else
 	$(call run_with_existing_containers, test-backend-integration-target)
 endif
 
+test-jobs-unit:
+	cd lumigator/python/mzai/jobs/evaluator_lite; \
+	uv run pytest -o python_files="evaluator_lite/tests/test_*.py"
```

Comment:
@ividal @aittalam @agpituk FYI, I'm adding additional unit test checks for the new jobs we're adding because they're not currently covered by the back-end checks. Right now they only cover evaluator_lite, since, if we expand the scope they will also test evaluator, which we are deprecating soon, and testing evaluator requires ray, which we don't want. So excluding to keep it simple.

Reply:
Addressed for now with https://github.com/mozilla-ai/lumigator/issues/559